cron_inactivity_period/models/ir_cron.py

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+# Copyright (C) 2018 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from openerp import api, fields, models
+
+
+_logger = logging.getLogger(__name__)
+
+
+class IrCron(models.Model):
+    _inherit = 'ir.cron'
+
+    inactivity_period_ids = fields.One2many(
+        comodel_name='ir.cron.inactivity.period', string='Inactivity Periods',
+        inverse_name='cron_id')
+
+    @api.model
+    def _callback(self, model_name, method_name, args, job_id):
```

Comment:
Sorry to ask here, but doesn't Odoo use the classic "_" prefix meaning that this method is not public and subject to change? In which case overriding would be dangerous and to be carefully tested (which it does not seem to be).

Reply:
> Sorry to ask here, but doesn't Odoo use the classic "_" prefix meaning that this method is not public 

Yes, this function will not be available via the API. (xml rpc call, etc...)

> and subject to change? 

No specially. And there are no changes when a version is stable. (in 8.0, there is no chance that the function changes, (excepted bug fixes))